test/node/module/authenticate/setClientCert.test.js

```diff
@@ -121,7 +121,7 @@ describe('Connection module', () => {
       const auth = new Auth()
         .setPasswordAuth({username: 'wrong_user', password: common.PASSWORD})
         .setClientCert(paramClientCert);
-      const conn = new Connection(common.DOMAIN, auth);
+      const conn = new Connection({domain: common.DOMAIN, auth: auth});
       const appModule = new App(conn);
```

Comment:
App module will be changed to single object params. Update App module first, then update this PR